### DISCUSSION TITLE
当前的QNN后端是否能够支持模型推理？

### DISCUSSION THREAD

**Discussion by FranzKafkaYu** [chraac/llama.cpp#3] [Q&A] [answered]
@chraac Hi，作者你好。想问一下当前仓库的代码是否已经支持模型推理，还是说只是实现了部分算子？我可以基于当前版本的QNN后端部署大模型吗

**Comment by chraac** [comment, marked as answer]
你好，现在matmul的这个算子还有问题，然后还有一个分支在重构中，具体可以看看这个PR(#2 )里面的说明，对代码有相关疑问也可以在里面讨论
